x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/esql/search_filter.cy.ts

```diff
@@ -43,51 +44,51 @@ describe.skip(
     beforeEach(() => {
       login();
       visitWithTimeRange(ALERTS_URL);
+      openActiveTimeline();
       createNewTimeline();
       goToEsqlTab();
       updateDateRangeInLocalDatePickers(DISCOVER_CONTAINER, INITIAL_START_DATE, INITIAL_END_DATE);
     });
 
-    it('should show data according to esql query', () => {
+    it('should show data according to the esql query', () => {
       addDiscoverEsqlQuery(`${esqlQuery} | limit 1`);
       submitDiscoverSearchBar();
       cy.get(DISCOVER_RESULT_HITS).should('have.text', 1);
     });
+
     it('should be able to add fields to the table', () => {
       addFieldToTable('host.name');
       addFieldToTable('user.name');
       cy.get(GET_DISCOVER_DATA_GRID_CELL_HEADER('host.name')).should('be.visible');
       cy.get(GET_DISCOVER_DATA_GRID_CELL_HEADER('user.name')).should('be.visible');
     });
 
-    context('navigation', () => {
-      it('should remove the query when back is pressed after adding a query', () => {
-        addDiscoverEsqlQuery(esqlQuery);
-        submitDiscoverSearchBar();
-        cy.get(DISCOVER_ESQL_INPUT_TEXT_CONTAINER).then((subj) => {
-          const currentQuery = subj.text();
-          const sanitizedQuery = convertNBSPToSP(currentQuery);
-          expect(sanitizedQuery).to.eq(esqlQuery);
-        });
-        cy.go('back');
-        cy.get(DISCOVER_ESQL_INPUT_TEXT_CONTAINER).then((subj) => {
-          const currentQuery = subj.text();
-          const sanitizedQuery = convertNBSPToSP(currentQuery);
-          expect(sanitizedQuery).to.not.eq(esqlQuery);
-        });
+    it('should remove the query when the back button is pressed after adding a query', () => {
+      addDiscoverEsqlQuery(esqlQuery);
+      submitDiscoverSearchBar();
+      cy.get(DISCOVER_ESQL_INPUT_TEXT_CONTAINER).then((subj) => {
+        const currentQuery = subj.text();
+        const sanitizedQuery = convertEditorNonBreakingSpaceToSpace(currentQuery);
+        expect(sanitizedQuery).to.eq(esqlQuery);
       });
+      cy.go('back');
```

Comment:
though it's just a one liner, I wonder if it would be cleaner to extract this into a function. It is kind of a `tasks` in a way?

Reply:
I get that, but I think it's best to have tasks be specific to the application or extend base cypress functionality in some way, so if we had a a go back, then do something, I think it would make sense, but going to leave it as is for now